lib/segment/src/data_types/vectors.rs

```diff
@@ -234,6 +248,14 @@ impl NamedVectorStruct {
             NamedVectorStruct::Named(v) => v.vector,
         }
     }
+
+    // TODO unify with NamedVectors::to_new_vector when exposing API
+    pub fn to_new_vector(self) -> Vector {
```

Comment:
It seems regular `Into` trait can be implemented here instead of explicit function